src/HatchIntakeSubsystem.cpp

```diff
@@ -26,32 +26,32 @@ void HatchIntakeSubsystem::robotInit(){
     m_hatchActuator.SetSensorPhase(true);
 }
 
-void HatchIntakeSubsystem::teleopInit(){
+void HatchIntakeSubsystem::teleopInit() {
    
 }
 
-void HatchIntakeSubsystem::teleop(){
+void HatchIntakeSubsystem::teleop() {
     // When the right trigger is pressed the intake actuates down and spins the intake wheels until it gets
     // a hatch then it actuates up.
-    if (operatorJoystick->GetRisingEdge(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER)){
-        if(m_hatchActuator.GetSelectedSensorPosition() >= m_actuatorTopLimit.Get() - m_hatchActuatorDeadband.Get()){
+    if (operatorJoystick->GetRisingEdge(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER)) {
+        if(m_hatchActuator.GetSelectedSensorPosition() >= m_actuatorTopLimit.Get() - m_hatchActuatorDeadband.Get()) {
             SetHatchActuatorDown();
         }
-        if(m_hatchActuator.GetSelectedSensorPosition() <= m_actuatorBottomLimit.Get() + m_hatchActuatorDeadband.Get()){
-            if(GetHatchState()){
+        if(m_hatchActuator.GetSelectedSensorPosition() <= m_actuatorBottomLimit.Get() + m_hatchActuatorDeadband.Get()) {
```

Comment:
This is not deadbanding the values correctly. In order to deadband you need to make sure that the absolute value of the difference between the desired position and the current position is less than the deadband range.